server/models/visits.js

```diff
@@ -81,11 +81,15 @@ var visits = {
                 fuzziness: 'AUTO',
                 operator: 'and',
                 fields: [
+                  'title',
                   'extractedTitle',
                   'extractedContent',
+                  'extractedLead',
                   'extractedDescription',
+                  'extractedUrl',
```

Comment:
The usefulness of this will really depend on tokenizer being used for this field.